flink-connector-oceanbase-cdc/src/test/java/com/ververica/cdc/connectors/oceanbase/table/OceanBaseConnectorITCase.java

```diff
@@ -382,15 +393,17 @@ public void testAllDataTypes() throws Exception {
                         + "    timestamp_c TIMESTAMP,\n"
```

Comment:
The type of `timestamp_c` should be `TIMESTAMP_LTZ`, if I don't misunderstand it.

Reply:
You're right. Fixed.

Reply:
I tried `TIMESTAMP_LTZ` here, and the timezone config for `tEnv` seems not work. Maybe it's not supported in `values` connector? Keep it for now.